integration/aggregate_documents_compat_test.go

```diff
@@ -101,11 +101,9 @@ func testAggregateStagesCompatWithProviders(tt *testing.T, providers shareddata.
 			for i := range targetCollections {
 				targetCollection := targetCollections[i]
 				compatCollection := compatCollections[i]
-				tt.Run(targetCollection.Name(), func(tt *testing.T) {
+				tt.Run(targetCollection.Name(), func(t *testing.T) {
```

Comment:
I'm not sure what is our desired state of using `t` and `tt`. For now I didn't put any thought in it.

Reply:
I think we should revert all `tt`s to `t` as we remove skip/failsFor functions and fields

Reply:
Let's replace all of the `tt` inside whole test function if `FailsForSQLite` is not used.